### DISCUSSION TITLE
tutorial files [Darcy_thermo_mech] not working

### DISCUSSION THREAD

**Discussion by majumdararn** [idaholab/moose#17544] [Q&A Getting Started] [answered]
I am new to moose. 
I tried to run the tutorial files of darcy_thermo_mech.
1) I ran the make file.
2) I ran the tests.
3) I went into problems directory and tried ../darcy_thermo_mech-opt -i step1.i  
4) I opened the result in ParaView. As per the result, the pressure value is zero everywhere. Could anyone help me with this? Please share if I need to share more information.

Note: 
1) My peacock is also not running. 
2) Everything is working fine in the examples folder. 
3) I have not changed anything in the input file. 
4) I activated moose. 
5) Even the tutorial files for steps 1 and 2 worked for the first time.

**Comment by cticenhour** [comment, marked as answer]
You may be looking at the initial condition, which is zero for that step (the default). MOOSE allows the definition of an initial condition, even for Steady-type problems. Look for a "Play" symbol button at the top of your ParaView window. It should look something like this: 

![Screen Shot 2021-04-08 at 8 10 45 AM](https://user-images.githubusercontent.com/16690058/114041699-fe334080-9841-11eb-9791-67343477887f.png)

Press that play button to advance to Timestep 1.

**Reply by majumdararn** [reply]
Thank you for your solution. It worked. However, peacock still does not work. Should I raise another question for that?

I explain the problem in any case.
If I want to open the result with peacock, I use the following command:
/moose/python/peacock/peacock -r step1_out.i
But, it shows the following error.
Automatic executable search disabled
Segmentation fault (core dumped)

If I want to open the input file with peacock, I use the following command:
/moose/python/peacock/peacock -i step1.i
In this case, peacock opens. For the first time when I ran the simulation within peacock it ran and then when I wanted to view the result. peacock closed suddenly.
The command line message including error is folllowing:
Running command: /home/amajumda/projects_moose/moose/tutorials/darcy_thermo_mech/step01_diffusion/darcy_thermo_mech-opt Outputs/csv=true -i peacock_run_exe_tmp_step1.i
Working directory: /home/amajumda/projects_moose/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems
Finished: Exit code: 0
Segmentation fault (core dumped)

From the sencond time onwards, peacock opens and the simulation don't run.
The command line error is the following:
Found executable: /home/amajumda/projects_moose/moose/tutorials/darcy_thermo_mech/step01_diffusion/darcy_thermo_mech-opt
Exodus filenames: ['peacock_run_exe_tmp_step1_out.e']
Segmentation fault (core dumped)

**Reply by cticenhour** [reply]
I'm glad it worked! Please open a new discussion question for your peacock issue.

**Reply by majumdararn** [reply]
Thanks. I opened a new issue. For future reference, if someone is trying to refer to this discussion, they can go to the following link for the peacock issue.
https://github.com/idaholab/moose/discussions/17560
